### PR TITLE
Allow env cluster size override

### PR DESCRIPTION
**Pull request by sam-github**
Should fix the test failures.

### CODE REVIEW

**Comment by rmg** [comment]
Doesn't fix the tests for me (or for CI), but does move the point of failure for me. Thanks to the improved debug output, I can see that at least one of the CI machines is hanging on `test/test-pmctl-local.js:175`.

**Comment by rmg** [comment]
Correction, the last `waiton` that CI ran on that run was at 175, so it got _at least_ that far before timing out.

**Comment by sam-github** [comment]
I just force pushed a test to test-server-metadata. I'm running all the tests again, but I'm not seeing what you are seeing. I'll try with deps from ci to see if that changes anything.

File: test/test-pmctl-local.js [hunk 1]
```diff
@@ -57,6 +57,7 @@ helper.pmctl.waiton = waiton;
 function waiton(cmd, output) {
   while (true) {
     try {
+      console.trace('WAITON %s', cmd);
```
Comment (rmg) [inline test/test-pmctl-local.js]: If this is moved to the first line of `expect` then we can see non-`waiton` calls as well. For extra convenience, also log the expected value.

**Pull request merged**